languages/python/meta.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Python 3.11",
```

Comment:
Maybe change this to just `Python`? I'm planning on adding support for codefort to automatically get the version with a new script in meta.json